src/ardrive.ts

```diff
@@ -290,11 +291,11 @@ export class ArDrive extends ArDriveAnonymous {
 
 		const originalFileMetaData = await this.getPrivateFile(fileId, driveKey);
 
-		if (destFolderDriveId !== originalFileMetaData.driveId) {
+		if (!destFolderDriveId.isEqualTo(originalFileMetaData.driveId)) {
 			throw new Error(errorMessage.cannotMoveToDifferentDrive);
 		}
 
-		if (originalFileMetaData.parentFolderId === newParentFolderId) {
+		if (`${originalFileMetaData.parentFolderId}` === `${newParentFolderId}`) {
```

Comment:
We cannot use `isEqualTo()` because parentFolderId is nullable, right?
```suggestion
		if (originalFileMetaData.parentFolderId.isEqualTo(newParentFolderId)) {
```